src/about-menu/about-menu.html

```diff
@@ -78,24 +78,19 @@
 <body> 
   <div>  
     <img src="./studio-app-icon.png" alt="Studio App Icon" width="64" height="64">
-        <p class="studio-name">Studio by WordPress.com</p>
-        <p class="version">Version <span id="version"></span></p>
+        <p class="studio-name" id="studio-by-wpcom">Studio by WordPress.com</p>
+        <p class="version"><span id="version-text">Version</span> <span id="version"></span></p>
         <p class="links">
           <a href="https://github.com/Automattic/studio" target="_blank">GitHub</a>
```

Comment:
I did not add translation for `GitHub` since it is a proper name. Happy to add it if it is considered necessary.